osbs/http.py

```diff
@@ -34,6 +41,22 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
+    def is_forced_retry(self, method, status_code, has_retry_after=False):
```

Comment:
Oh, right, it was renamed - https://github.com/shazow/urllib3/blob/master/urllib3/util/retry.py#L294